freedomofpress/securedrop
# Dev docs: Explains libvirt cleanup procedures

## Status

Ready for review.

## Description of Changes

Refs #4993. May not resolve, pending further investigation of root cause.

Updates developer documentation with specific steps for managing the staging environment VMs on Debian hosts.

## Testing

* Check out this branch. 
* Run `make clean` (be aware this will destroy all non-version-controlled files, including config)
* Run `make staging`, confirm no errors.
* Run `make clean` again after staging run, confirm no errors.

If you encounter errors, try to investigate and determine where exactly the failure occurred. For instance, was it during VM removal? Try setting `MOLECULE_DEBUG=1` and see if that yields more useful information. 

## Deployment
No concerns for production deployments, these changes are dev-env only.

## Checklist

### If you made non-trivial code changes:


### If you made changes to documentation:
